lib/install/validate/network.go

```diff
@@ -317,6 +317,9 @@ func (v *Validator) checkVDSMembership(ctx context.Context, network types.Manage
 		return nil
 	}
 
+	if v.Session.Cluster == nil {
+		return errors.New("cluster not set")
```

Comment:
cluster is not a value set by user, might have more message, suggest user to check compute-resource settings.